packages/iov-socket/src/queueingstreamingsocket.ts

```diff
@@ -3,17 +3,27 @@ import { Stream } from "xstream";
 // tslint:disable-next-line:no-submodule-imports
 import concat from "xstream/extra/concat";
 
+import { DefaultValueProducer, ValueAndUpdates } from "@iov/stream";
+
 import { SocketWrapperMessageEvent } from "./socketwrapper";
 import { StreamingSocket } from "./streamingsocket";
 
 function wrapStream<T>(stream: Stream<T>): Stream<T> {
   return concat(stream.replaceError(Stream.never), Stream.never());
 }
 
+export enum ConnectionStatus {
```

Comment:
this new type (and QueueingStreamingSocket as well) should be exported in index.ts